sktime/performance_metrics/annotation/metrics.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Metrics for evaluating performance of segmentation estimators.
+
+Metrics are suitable for comparing predicted change point sets
+against true change points and quantify the error.
+"""
+
+import numpy as np
+import numpy.typing as npt
+from scipy.spatial.distance import directed_hausdorff
+from sklearn.utils import check_array
+
+__author__ = ["lmmentel"]
+
+
+def annotation_error(
```

Comment:
is this the standard name for this? the phrase 'annotation error' sounds like any potential way of measuring error between annotations.

Reply:
This name was used in one of the papers, can't remember which one. It is used in a specific context so I wouldn't say it's a big deal. Happy to accept a suggestion on what to change it to.

Reply:
some options:
- 'count_error'
- 'counting_error'
- 'number_error'
- 'number_changepoints_error'
- 'difference_in_count'...